QRScannerTests/TestCases/QRScannerViewTests.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  QRScannerViewTests.swift
+//  QRScannerTests
+//
+//  Created by wbi on 2019/10/16.
+//  Copyright © 2019 mercari.com. All rights reserved.
+//
+
+import XCTest
+@testable import QRScanner
+
+class QRScannerViewTests: XCTestCase {
+
+    var qrScanner: QRScannerView!
+    var delegate: QRScannerSpyDelegate!
+
+    override func setUp() {
+        super.setUp()
+        qrScanner = QRScannerView()
+        delegate = QRScannerSpyDelegate()
+    }
+
+    override func tearDown() {
+        super.tearDown()
+    }
+
+    func testConfigure() {
```

Comment:
This test is sample.
For device simulator, `AVCaptureDevice.default(.video)` is always failed.

Of course we should test `didSuccess` and `didChangeTorchActive` so we will create mocks of `AVCaptureDevice` or so on.